src/config/project.rs

```diff
@@ -285,7 +296,7 @@ fn default_pkg_dir() -> Utf8PathBuf {
 }
 
 fn default_site_root() -> Utf8PathBuf {
-    Utf8PathBuf::from("target").join("site")
+    Utf8PathBuf::from("CARGO_TARGET_DIR").join("site")
```

Comment:
Add synonym, too https://doc.rust-lang.org/cargo/reference/config.html#buildtarget-dir

Reply:
@aperepel This function only instantiates a _single_ default, adding a synonym/second path variant doesn't make sense here (to me). Did you mean that it should simply be replaced with `"CARGO_BUILD_TARGET_DIR"` (or the appropriate constant)?